atom/browser/ui/tray_icon_cocoa.mm

```diff
@@ -75,6 +76,8 @@ - (void)updateDimensions {
 }
 
 - (void)removeItem {
+  // Turn off tracking events to prevent crash
+  [self removeTrackingArea:trackingArea_];
```

Comment:
We need to

1. Release the trackingArea_ pointer
2. Safeguard against multiple calls

```
if (trackingArea_) {
  [self removeTrackingArea:trackingArea_];
  trackingArea_.reset();
}